webapp/channels/src/components/onboarding_tasklist/onboarding_tasklist_completed.tsx

```diff
@@ -239,7 +242,8 @@ const Completed = (props: Props): JSX.Element => {
                                     link: (msg: React.ReactNode) => (
                                         <ExternalLink
                                             location='onboarding_tasklist_completed'
-                                            href='https://mattermost.com/download/#desktop'
+                                            href='https://mattermost.com/download'
+                                            anchor='#desktop'
```

Comment:
nit: Maybe the `#` should be appended in the `ExternalLink` component when `anchor` prop is provided.